### ISSUE TITLE
Include the core definitions by default

### ISSUE THREAD

**Issue by bradennapier** [codemix/flow-runtime#133]
It is pretty obnoxious to have to manually include the core definitions one-by-one that are apart of flow itself.  For example "iterable" or "Promise" do not work unless you manually include them.  The tool you mention to help with that actually does not work at all for me and/or i had major issues getting it to output a single thing (others reported the same thing).  

Perhaps simply automatically including the core libraries?

https://github.com/facebook/flow/tree/master/lib

**Comment by bradennapier** [comment]
Ahh, it isn't compatible.  Makes more sense.

```

client?e4ca:119 ./internals/flow/definitions/core.flow.js
Module build failed: SyntaxError: Unexpected token (200:4)

  198 |  */
  199 | declare class $ReadOnlyArray<+T> {
> 200 |     @@iterator(): Iterator<T>;
      |     ^
  201 |     toLocaleString(): string;
  202 |     // concat creates a new array
  203 |     concat<S, Item: $ReadOnlyArray<S> | S>(...items: Array<Item>): Array<T | S>;

 @ ./app/require-flow.dev.js 1:0-50
 @ multi (webpack)-dev-server/client?http://localhost:1212 webpack/hot/dev-server react-hot-loader/patch webpack-dev-server/client?http://localhost:1212/ webpack/hot/only-dev-server regenerator-runtime/runtime ./app/require-flow.dev.js ./app/index.js

```

**Comment by phpnode** [comment]
Sorry for the delay, the issue with $await should now be fixed in v0.15.0. As a babel plugin we don't have enough information about the whole codebase to include the necessary builtins automatically, as babel only gives us one file at a time. Therefore we need a webpack loader, which we're tracking in #165
